packages/form-js-viewer/src/util/index.js

```diff
@@ -60,4 +60,43 @@ export function generateIdForType(type) {
  */
 export function clone(data, replacer) {
   return JSON.parse(JSON.stringify(data, replacer));
+}
+
+/**
+ * Parse the schema for input variables a form might make use of
+ *
+ * @param {any} schema
+ *
+ * @return {string[]}
+ */
+
+export function getSchemaVariables(schema) {
```

Comment:
Will this be available via 

```js
import { getSchemaVariables } from '@bpmn-io/form-js';
```

Let's double check this, cf. [`form-js` exports](https://github.com/bpmn-io/form-js/blob/develop/packages/form-js/src/viewer.js).

Reply:
It is 👍

Reply:
When I import it directly from `@bpmn-io/form-js`, e.g. in the Playground, it throws

```js
import { getSchemaVariables } from '@bpmn-io/form-js';

console.log(getSchemaVariables(schema));
```

![image](https://user-images.githubusercontent.com/9433996/181748114-0f4b684b-d03a-428a-bc3f-9db9d5bb7b72.png)

But maybe it's just a setup problem on my side.

Reply:
Did you rebuild?

Reply:
Wait let me try this in the playground.

Reply:
Whoops no you're right, having issues too.

Reply:
@pinussilvestrus Should be fixed.